mlir/include/mlir/Dialect/AMDGPU/Transforms/Passes.td

```diff
@@ -37,10 +37,17 @@ def OptimizeSharedMemory : Pass<"amdgpu-optimize-shared-memory"> {
     attempts to optimize reads/writes from a memref representing GPU shared
     memory in order to avoid bank conflicts.
   }];
-
   let dependentDialects = [
     "memref::MemRefDialect", "vector::VectorDialect"
   ];
+  let options = [
+    Option<"sharedMemoryLineSizeBytes", "shared-memory-line-size-bytes", "uint64_t",
```

Comment:
Would be nice if we had one source of truth for the pass and transform dialect op, but this is fine for now.